orderapi/src/main/java/fi/hel/verkkokauppa/order/api/data/transformer/RefundAccountingTransformer.java

```diff
@@ -0,0 +1,23 @@
+package fi.hel.verkkokauppa.order.api.data.transformer;
+
+import fi.hel.verkkokauppa.order.api.data.accounting.RefundAccountingDto;
+import fi.hel.verkkokauppa.order.model.accounting.RefundAccounting;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RefundAccountingTransformer implements ITransformer<RefundAccounting, RefundAccountingDto> {
```

Comment:
FYI: You could have used https://github.com/City-of-Helsinki/verkkokauppa-core/blob/074e5ccba124b875c2dd321ee47ec4d893eba0af/common/src/main/java/fi/hel/verkkokauppa/common/mapper/AbstractModelMapper.java#L75 in here, but not needed for now because this works.